gpustack/utils/config.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Dict, get_type_hints, get_args, get_origin
+
+from fastapi import Request
+from transformers.hf_argparser import string_to_bool
+
+from gpustack.config.config import Config
+
+
+WHITELIST_CONFIG_FIELDS = {
+    "debug",
+    "system_default_container_registry",
```

Comment:
What is the value for it when it's the auto-detect quay.io case. Does it help https://github.com/gpustack/gpustack/issues/3159#issuecomment-3535439920?